docs/manual/pages/create/create_rondes.md

```diff
@@ -24,9 +24,9 @@ alsook op de kaart (5).
 
 De rondes hoeven niet in volgorde toegevoegd te worden.
 De volgorde kan gewijzigd worden door op de blauwe pijlen te drukken.
-In de finale volgorde is het gebouw dat boven de andere staat ook het eerste gebouw in de ronde.
+In de finale volgorde is het gebouw dat boven de anderen staat ook het eerste gebouw in de ronde.
 Via de indices die bij de gebouwen op de kaart staan, wordt dit ook duidelijker.
-Wanneer een verkeerd gebouw werd toegevoegd, kan deze verwijderd worden door op het rode kruisje te drukken.
+Wanneer een verkeerd gebouw werd toegevoegd, kan deze verwijderd worden door op het rood kruisje te drukken.
```

Comment:
Dit moet toch het rode kruisje zijn.
[zie link](https://www.vlaanderen.be/taaladvies/taaladviezen/bijvoeglijke-naamwoorden-verbuigen-verbuiging-met-of-zonder-e)

Reply:
Ah ja oeps, je mag het eventueel zelf aanpassen. Anders doe ik het straks wel.